boards/nrf52840dongle/Makefile.include

```diff
@@ -0,0 +1,27 @@
+# This board uses the vendor's serial bootloader
+
+PROGRAMMER ?= nrfutil
+
+ifeq (nrfutil,$(PROGRAMMER))
+
```

Comment:
I think you could move the

    FEATURES_REQUIRED += bootloader_nrfutil

here.

Reply:
Good point, fixed.

Reply:
Sorry, drop that

```Make
ifeq (,$(filter stdio_cdc_acm,$(USEMODULE)))
  # The arduino and nrfutil bootloader features cannot be used if the
  # stdio_cdc_acm module is not used
  FEATURES_BLACKLIST += bootloader_arduino
  FEATURES_BLACKLIST += bootloader_nrfutil
endif
```

Reply:
Please don't move this in a Makefile.include file !!!!